src/averageAges.js

```diff
@@ -15,11 +15,23 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+  let sumOfMenAges;
+  let averageManAge;
+  const menDiedInGivenCentury = men.filter(person => (
+    Math.ceil(person.died / 100) === century
+  ));
```

Comment:
what if a century is not passed? You still create this variable. But let's try to get the needed men right inside the filter callback